fprime-vxworks/VxWorks/Log.cpp

```diff
@@ -11,6 +12,10 @@ namespace Os {
         this->registerLogger(this);
     }
 
+
+    static char logBuffers[FW_LOG_TEXT_BUFFER_DEPTH][FW_LOG_TEXT_BUFFER_SIZE];
```

Comment:
One more comment:

`FW_LOG_TEXT_BUFFER_DEPTH` seems to be undefined. I looked through the fprime repo and could not find it.

Reply:
I put it in the `FpConfig.hpp` for Sabertooth:

```
// Define the depth of the Os::Logger circular buffer - may not be used on all platforms
#ifndef FW_LOG_TEXT_BUFFER_DEPTH
#define FW_LOG_TEXT_BUFFER_DEPTH              100   //!< Max size of string for text log message
#endif
```

The idea is that a project could define the depth of the FIFO depending on their memory requirements.

Reply:
Gotcha. Thank you. LGTM